### PR TITLE
Update testbed to use OTLP by default

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: testbed/testbed/test_case.go [hunk 1]
```diff
@@ -121,7 +122,7 @@ func NewTestCase(
 	require.NoErrorf(t, os.MkdirAll(tc.resultDir, os.ModePerm), "Cannot create directory %s", tc.resultDir)
 
 	// Set default resource check period.
-	tc.resourceSpec.ResourceCheckPeriod = 3 * time.Second
+	tc.resourceSpec.ResourceCheckPeriod = 1 * time.Second
```
Comment (tigrannajaryan) [inline testbed/testbed/test_case.go:125]: This an cause problems during startup when we have a spike of CPU consumption. Why is this change needed?

Comment (bogdandrutu) [reply testbed/testbed/test_case.go]: Reverted, was looking for error cause.

**Comment by tigrannajaryan** [comment]
Build failed.
